Sources/Commands/SwiftTestTool.swift

```diff
@@ -205,6 +211,11 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
             let testSuites = try getTestSuites(path: testPath)
             let tests = testSuites.filteredTests(specifier: options.testCaseSpecifier)
 
+            // if user called --num-workers without the --parallel option, throw an error
+            if (options.numberOfWorkers>0) && !options.shouldRunInParallel {
```

Comment:
We should also validate that options.numberOfWorkers is never 0 or less. Two nits:

- Prefer one space in both sides of the operator, i.e. use `a > b` and not a>b
- Avoid using parenthesis in if statements